### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by alicera** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#177]
I train the project with pytorch 1.7.0

It show the below message,
Should I exchange the scheduler.step() (line 77) and optimizer.step() (line 106)?

/opt/conda/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:136: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
Iter: 100

**Comment by Daniil-Osokin** [comment]
Hi! You may do it.

**Comment by alicera** [comment]
Thanks, I got it!
